appveyor.yml

```diff
@@ -15,11 +17,14 @@ install:
   - ps: wget 'https://s3.amazonaws.com/kylo-pl-bucket/yaml-cpp-0.5.1-x86_64_mingw-w64_4.8.3_win32_seh.7z' -OutFile "$pwd\yaml-cpp.7z"
   - ps: 7z.exe x yaml-cpp.7z -oC:\tools | FIND /V "ing  "
 
+  - ps: wget 'https://github.com/puppetlabs/leatherman/releases/download/%LEATHERMAN_VERSION%/leatherman.7z' -OutFile "$pwd\facter.7z"
```

Comment:
This is a powershell command, `%LEATHERMAV_VERSION%` is the wrong syntax. You should use double quotes and `$LEATHERMAN_VERSION`.